feat: hide & show measurements (#2651)
cognitedata/reveal: documentation/docs/examples/point-to-point-measurement.mdx

}, 6000);
```

### Hide and show measurements

> It's possible to toggle between hide & show the measurements in the viewer.
> It can be useful to view the details of an object without measurements blocking the view.

Reviewer: ```suggestion
It may be useful to view the details of an object without measurements blocking the view.
It's possible to toggle measurement visibility on and off in the viewer.
```
